app/grandchallenge/discussion_forums/models.py

```diff
@@ -0,0 +1,120 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django_extensions.db.fields import AutoSlugField
+
+from grandchallenge.core.models import UUIDModel
+
+
+class TopicTypeChoices(models.TextChoices):
+    DEFAULT = "DEFAULT", "Default topic"
+    STICKY = "STICKY", "Sticky topic"
+    ANNOUNCE = "ANNOUNCE", "Announcement topic"
+
+
+class Forum(UUIDModel):
+    # name gets populated from challenge title, so use same max_length
+    name = models.CharField(max_length=60)
+    slug = AutoSlugField(populate_from="name", max_length=60)
```

Comment:
Quick thought - does the forum need a name and slug? We could limit it to one forum per challenge/parent object (one to one related) and have it accessible at `/forum/`. I don't think that we have ever seen the need for multiple forums and it is cleaner.

Reply:
Good point. We used the slug previously for the urls, but since we're changing those anyway, I think I agree that the slug and name are redundant. The relationship between forum and challenge is already OneToOne and I indeed don't foresee the need for multiple forums in the future either. 

Worst case, if a name and slug became necessary at some point, it also wouldn't be too hard to add in retrospect since we're anyway auto-populating those fields from the challenge short_name.